sw/device/silicon_creator/rom_ext/e2e/attestation/print_certs.c

```diff
@@ -0,0 +1,82 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+#include "sw/device/silicon_creator/lib/drivers/flash_ctrl.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+const char base64[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+void base64_encode(char *dest, const uint8_t *data, int32_t len) {
```

Comment:
```suggestion
static void base64_encode(char *dest, const uint8_t *data, int32_t len) {
```
?

Same throughout?

Reply:
Fixed.